.trunk/trunk.yaml

```diff
@@ -16,6 +16,7 @@ runtimes:
 
 lint:
   enabled:
+    - checkov@2.3.75
```

Comment:
We don't need this enabled ourselves do we?

Reply:
no, removed